app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+  include TasksHelper
+
+  skip_before_action :set_project, unless: -> { current_project }
+  skip_before_action :set_nodes, unless: -> { current_project }
+
+  def index
+    @default_columns = ['Title', 'Methodology', 'List', 'Due Date', 'Assigned']
+
+    if current_project
```

Comment:
This seems to be a smell? Reusing a controller for in/out of project context?

Reply:
refactored to be in project always in CE